docs/tutorials/tabular/tabular-quick-start.ipynb

```diff
@@ -0,0 +1,345 @@
+{
```

Comment:
This quick start is pretty good. We may want to consider using titanic instead of this example (although we definitely should keep this example for at least 1 tutorial, since it is quite good). For the purposes of an initial merge, I am ok with this tutorial.

The main thing we should add to this tutorial is a comparison of how simple this solution is + how strong the result is compared to DeepMind's original solution that was on the cover of nature (300+ lines of code & 80% accuracy with DeepMind vs 3 lines of code & 95% accuracy with AutoGluon). This comparison shouldn't be present in a quick-start tutorial, which is why we should wait to add that comparison until we make a different quick-start tutorial for Tabular.